rasa/core/nlg/callback.py

```diff
@@ -52,9 +54,18 @@ class CallbackNaturalLanguageGenerator(NaturalLanguageGenerator):
     json. The generator will use this message to create a response for
     the bot."""
 
-    def __init__(self, endpoint_config: EndpointConfig) -> None:
+    def __init__(
+        self,
+        endpoint_config: EndpointConfig,
+        responses: Dict[Text, List[Dict[Text, Any]]],
```

Comment:
Same here, the modification of this method signature with a positional argument constitutes a breaking change if I'm not mistaken, we should add it as a default keyword arg, e.g. `responses: Optional[Dict[..]] = None`